src/MLJTreeParzen.jl

```diff
@@ -217,6 +217,22 @@ get_trialhist(history) =
         completed_trial
     end
 
+function update_param!(model, param, val)
+    if param in propertynames(model)
```

Comment:
Do we need to throw an error if the model doesn't have that param? Something like this 
`            if hasproperty(model, param)
                setproperty!(model, param, value)
            else
                error("Invalid param: $param")
            end`

Reply:
We can have own error but it's not needed because the unsupported params are usually very clear for different models. It's more about if we want to make them more consistent within TreeParzen. I don't think we do it elsewhere.

Reply:
I updated model parameter error and added test with the expected message. However, this change affects the existing DummyModel which before this change could get away with some random parameters and now it can't 😸 . It's because with real models, this unsupported parameter error would normally come from the model library but clearly this one wasn't returning any of that.